src/platform_impl/linux/window.rs

```diff
@@ -139,22 +140,20 @@ impl Window {
       window.move_(x, y);
     }
 
-    // Set Transparent
-    if attributes.transparent {
-      if let Some(screen) = window.screen() {
-        if let Some(visual) = screen.rgba_visual() {
-          window.set_visual(Some(&visual));
-        }
+    // Set GDK Visual
+    if let Some(screen) = window.screen() {
+      if let Some(visual) = screen.rgba_visual() {
+        window.set_visual(Some(&visual));
       }
+    }
 
-      window.connect_draw(|_, cr| {
-        cr.set_source_rgba(0., 0., 0., 0.);
-        cr.set_operator(cairo::Operator::Source);
-        let _ = cr.paint();
-        cr.set_operator(cairo::Operator::Over);
-        Inhibit(false)
-      });
-      window.set_app_paintable(true);
+    // Set a few attributes to make the window can be painted.
+    // See Gtk drawing model for more info:
+    // https://docs.gtk.org/gtk3/drawing-model.html
+    window.set_app_paintable(true);
+    let widget = window.upcast_ref::<gtk::Widget>();
+    unsafe {
+      gtk::ffi::gtk_widget_set_double_buffered(widget.to_glib_none().0, 0);
```

Comment:
Many are optional but I make these become default to all windows.
Tested with wry and it looks fine.
This should also reduce some overhead when dealing with draw signal overall.

Reply:
sounds fine, we can revert back if any issues are reported.